pandapower/opf/run_powermodels.py

```diff
@@ -33,29 +34,36 @@ def _runpm(net, delete_buffer_file=True):  # pragma: no cover
     # writes pm json to disk, which is loaded afterwards in julia
     buffer_file = dump_pm_json(pm)
     # run power models optimization in julia
-    result_pm = _call_powermodels(buffer_file, net._options["julia_file"])
+    result_pm = _call_powermodels(buffer_file, net._options["julia_file"], sysimage_file)
     # read results and write back to net
     read_pm_results_to_net(net, ppc, ppci, result_pm)
     if delete_buffer_file:
         # delete buffer file after calculation
         os.remove(buffer_file)
 
 
-def _call_powermodels(buffer_file, julia_file):  # pragma: no cover
+def _call_powermodels(buffer_file, julia_file, sysimage_file):  # pragma: no cover
     # checks if julia works, otherwise raises an error
     try:
         import julia
-        from julia import Main
+        from julia.api import Julia
     except ImportError:
         raise ImportError("Please install pyjulia to run pandapower with PowerModels.jl")
     try:
-        j = julia.Julia()
+        is_conda = Path.exists(Path(sys.prefix) / 'conda-meta')
+        if sysimage_file:
```

Comment:
and then: `if sysimage_file is not None:` and check also with `pathlib.Path.isfile()`  if the file exists